src/command/template/regenerate_queries.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Install this CLI with `cargo install graphql_client_cli`
+graphql-client generate --schema-path schema.graphql queries.graphql \
+  --response-derives 'Debug,Serialize,PartialEq,Eq,Clone' \
+  --custom-scalars-module crate::command::template::custom_scalars
```

Comment:
this is something we'd likely want to build into the `cargo xtask prep` command instead of a shell script if we were to add this tool to the toolchain

Reply:
We wouldn't need this to run unless the queries had changed—is there a way in `xtask` to auto-regen if needed but only if needed?

I was thinking about doing this in a `build.rs` somehow.

Reply:
not any way to do it in the code that exists now but i'm sure there's a heuristic we could use for it. i'm a bit worried about having a hybrid approach where some graphql queries use this and some use the other style - i'd like it to be consistent if we can help it to reduce cognitive overhead when working with this code.

Reply:
I agree that consistency would be ideal—and we should choose the approach we want to use and work toward it. Just not sure that changing all of the code in one fell swoop is feasible—maybe I'm overestimating the number of queries there are?

I'd be happy to do follow-up PR(s) converting other queries to this style if we prefer it, or I can convert this to the other style if we like that one better. I just don't want to undo and redo this if we're eventually moving in this direction anyway 😛.